async/Mailbox_test.go

```diff
@@ -58,7 +58,7 @@ func storeValue_withMailbox(num int) error {
 			successfulWrites++
 		}
 		returnedWrites++
-		fmt.Println("completedWrites", returnedWrites)
+		log.Println("completedWrites", returnedWrites)
```

Comment:
do we want to convert all log.Print to log.Info?  (I also saw one in sagaGenerators... not sure if there are more lurking out there.